code/modules/vehicles/mecha/mecha_construction_paths.dm

```diff
@@ -1072,138 +754,13 @@
-			else
-				user.visible_message(span_notice("[user] disengages the bluespace crystal."), span_notice("You disengage the bluespace crystal."))
-		if(19)
-			if(diff==FORWARD)
-				user.visible_message(span_notice("[user] secures the power cell."), span_notice("You secure the power cell."))
-			else
-				user.visible_message(span_notice("[user] pries the power cell from [parent]."), span_notice("You pry the power cell from [parent]."))
-		if(20)
-			if(diff==FORWARD)
-				user.visible_message(span_notice("[user] installs the phase armor layer to [parent]."), span_notice("You install the phase armor layer to [parent]."))
-			else
-				user.visible_message(span_notice("[user] unfastens the power cell."), span_notice("You unfasten the power cell."))
-		if(21)
-			if(diff==FORWARD)
-				user.visible_message(span_notice("[user] secures the phase armor layer."), span_notice("You secure the phase armor layer."))
-			else
-				user.visible_message(span_notice("[user] pries the phase armor layer from [parent]."), span_notice("You pry the phase armor layer from [parent]."))
-		if(22)
-			if(diff==FORWARD)
-				user.visible_message(span_notice("[user] welds the phase armor layer to [parent]."), span_notice("You weld the phase armor layer to [parent]."))
-			else
-				user.visible_message(span_notice("[user] unfastens the phase armor layer."), span_notice("You unfasten the phase armor layer."))
-		if(23)
-			if(diff==FORWARD)
-				user.visible_message(span_notice("[user] installs [I] to [parent]."), span_notice("You install [I] to [parent]."))
-			else
-				user.visible_message(span_notice("[user] cuts phase armor layer from [parent]."), span_notice("You cut the phase armor layer from [parent]."))
-		if(24)
-			if(diff==FORWARD)
-				user.visible_message(span_notice("[user] secures Phazon Armor Plates."), span_notice("You secure Phazon Armor Plates."))
-			else
-				user.visible_message(span_notice("[user] pries Phazon Armor Plates from [parent]."), span_notice("You pry Phazon Armor Plates from [parent]."))
-		if(25)
-			if(diff==FORWARD)
-				user.visible_message(span_notice("[user] welds Phazon Armor Plates to [parent]."), span_notice("You weld Phazon Armor Plates to [parent]."))
-			else
-				user.visible_message(span_notice("[user] unfastens Phazon Armor Plates."), span_notice("You unfasten Phazon Armor Plates."))
-		if(26)
-			if(diff==FORWARD)
-				user.visible_message(span_notice("[user] carefully inserts the bluespace anomaly core into [parent] and secures it."),
-					span_notice("You slowly place the bluespace anomaly core into its socket and close its chamber."))
+	// We've already advanced the message step in ..(), so we have to offset by one to make sure we're on the right message
+	if((message_step - 1) == INSERT_ANOMALY_CORE_STEP)
+		if(diff==FORWARD)
+			user.balloon_alert_to_viewers("inserted and secured bluespace anomaly core")
```

Comment:
Might be a little lengthy for a `balloon_alert_to_viewers`

Reply:
Yeah... it's kind of two steps in one. I think chopping off the "and secured" would be easiest.